xml_inscriptions/metadata_only/PA.Phil.UP.UM.ETR.MS34200.xml

```diff
@@ -0,0 +1,164 @@
+                                </support>
+                                <condition ana="#complete.fragments">
+                                    <p/>
+                                </condition>
+                            </supportDesc>
+                            <!--  *************************************  <layoutDesc>  ********************************* -->
+                            <layoutDesc>
+                                <layout columns="0" writtenLines="0">
+                                    <p/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+
+                        <!--  *************************************  <handDesc>  ********************************* -->
+
+                        <handDesc hands="1">
+                            <handNote ana="#impressed.inscribed"> </handNote>
+                        </handDesc>
+
+                        <!--  *************************************  <decoDesc>  *********************************     -->
+                        <decoDesc>
+                            <decoNote>
+                                <p/>
+                            </decoNote>
+                        </decoDesc>
+                    </physDesc>
+
+                    <!--  *************************************  <history>  *********************************  -->
+
+                    <history>
+
+                        <summary>
+                            <rs/>  <!-- content can be "fake" or blank -->
+                        </summary>
+
+                        <origin>
+                            <date notBefore="-0551" notAfter="-0549">550 BC</date>
+                            <placeName ref="europe.italy">Orvieto</placeName>
+                        </origin>
+
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#"/>
+                        <biblScope unit="page">00</biblScope>
+                    </bibl>
+                    <bibl>
+                        <ref type="external"
+                            target="https://www.penn.museum/collections/object/259676">External
+                            Reference on the Penn Museum Collection</ref>
+                    </bibl>
+                    <bibl>
+                        <ref type="external"
+                            target="https://books.google.com/books?id=1_D8zqmCMWsC&amp;pg=PA165&amp;lpg=PA165&amp;dq=the+%22etruscan%22+museum+in+the+museum+bates&amp;source=bl&amp;ots=e_gF6ugH_V&amp;sig=ACfU3U2u10rPXmRI5EpfRlxx2k43Ie0kpg&amp;hl=en&amp;ppis=_e&amp;sa=X&amp;ved=2ahUKEwiF6cuHh4jmAhXmt1kKHdeIAC4Q6AEwAHoECAgQAQ#v=onepage&amp;q=the%20%22etruscan%22%20museum%20in%20the%20museum%20bates&amp;f=false"
```

Comment:
This seems to be another book? Send this to me for the bibliography instead of linking to the Google Books page.